goosebit/api/v1/software/routes.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
```

Comment:
Why are these lines needed?

Reply:
Not really sure why I've been adding them, just a good failsafe.  Need to modify my IDE settings to stop complaining about them I think.